float32 DesiredPitch
float32 DesiredYaw
